lib/ui/views/emergency_view.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:notredame/core/viewmodels/emergency_viewmodel.dart';
+import 'package:notredame/generated/l10n.dart';
+import 'package:notredame/ui/utils/app_theme.dart';
+import 'package:stacked/stacked.dart';
+import 'package:webview_flutter/webview_flutter.dart';
+
+class EmergencyView extends StatefulWidget {
+  final String title;
+  final String description;
+  const EmergencyView(this.title, this.description);
+
+  @override
+  _EmergencyViewState createState() => _EmergencyViewState();
+}
+
+class _EmergencyViewState extends State<EmergencyView> {
+  WebViewController _controller;
+  @override
+  Widget build(BuildContext context) =>
+      ViewModelBuilder<EmergencyViewModel>.reactive(
+        viewModelBuilder: () => EmergencyViewModel(),
+        builder: (context, model, child) => Scaffold(
+          appBar: AppBar(title: Text(widget.title)),
+          floatingActionButtonLocation:
+              FloatingActionButtonLocation.centerFloat,
+          floatingActionButton: FloatingActionButton.extended(
+            onPressed: () {
+              model.makePhoneCall(
+                  'tel:${AppIntl.of(context).security_emergency_number}');
```

Comment:
You should handle the possible error. In the case, the phone application is not available an exception is thrown and so the application will crash, we should display a Toast or a snackbar to handle the error 😄 (you can look here to have an example)

Reply:
done